filehandling.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+
+struct employee
+{
+	char employeeName[50];
+	int employeeId;
+};
+
+int main()
+{
+	FILE *fp=NULL;
+	struct employee emp[2];
+	int i;
+	for(i=0;i<2;i++)
+{
+	printf("Enter the employee details\n");
+	printf("Name:");
+	scanf("%s",emp[i].employeeName);
+	printf("Id:");
+	scanf("%d",&emp[i].employeeId);
+}
+
+fp=fopen("file.bin","wb");
+if(fp==NULL)
+{
+	printf("error in open file in write mode");
+}
+else
+{
+	for(i=0;i<2;i++)
+ fwrite(&emp[i],sizeof(struct employee),1,fp);
+}
+
+fp=fopen("file.bin","rb");
+{
```

Comment:
'{' should be inside if statement

Reply:
Indentation Issue:  try to use same format for each lines, either use Tab or Space. Don't use both in same code.
Also make code follow same format for each line inside a scope.
some thing like this:
```
if (fp == NULL) {  
    printf("error in open file in write mode");
} else {
    for (i = 0; i < 2; i++) {
         fwrite(&emp[i], sizeof(struct employee), 1, fp);
    }
}

fp = fopen("file.bin", "rb");
{
```

Reply:
I changed as your review commands . can you please verify once